includes/admin/settings/class-settings-advanced.php

```diff
@@ -137,7 +137,7 @@ public function get_settings() {
 							/* translators: %s: about page URL */
 							'desc'          => sprintf(
 								wp_kses(
-									__( 'Disable this option if you would like to disable the <a href="%s" target="_blank">GiveWP Setup page</a> that displays when GiveWP is first installed.', 'give' ),
+									__( 'This option controls the display of the <a href="%s" target="_blank">GiveWP Setup page</a> displayed when GiveWP is first installed.', 'give' ),
```

Comment:
I think the "display of the... displayed when" is redundant.

Reply:
agreed. removed the second one.